[Example]update NER example readme on module prediction (#13184)
apache/incubator-mxnet: example/named_entity_recognition/README.md

To train the model:

- `$ cd src && python ner.py`

To inference using trained model:

1. Re-create the bucketing module using `sym_gen` defined in `ner.py`
2. Loading saved parameters using `module.set_params()`

> Refer to [Bucketing Module example](https://github.com/apache/incubator-mxnet/blob/e9a590fa6554231fba404dad08acee5cd3e786a8/example/rnn/bucketing/cudnn_rnn_bucketing.py#L167) and this [issue](https://github.com/apache/incubator-mxnet/issues/5008) on Bucketing Module Prediction

Reviewer: can you reference this relative to the master rather than a particular commit?

> Refere to [Bucketing Module example](https://github.com/apache/incubator-mxnet/blob/e9a590fa6554231fba404dad08acee5cd3e786a8/example/rnn/bucketing/cudnn_rnn_bucketing.py#L167) and this [issue](https://github.com/apache/incubator-mxnet/issues/5008) on Bucketing Module Prediction

Reviewer 2: Nit : Typo in 'Refere'.

To train the model:

- `$ cd src && python ner.py`

> To inference using trained model:

Reviewer 3: ```suggestion
To run inference using trained model:
```

> 1. Re-create the bucketing module using `sym_gen` defined in `ner.py`

Reviewer 3: ```suggestion
1. Recreate the bucketing module using `sym_gen` as defined in `ner.py`
```

2. Loading saved parameters using `module.set_params()`

> Refer to [Bucketing Module example](https://github.com/apache/incubator-mxnet/blob/master/example/rnn/bucketing/cudnn_rnn_bucketing.py#L167)

Reviewer 3: ```suggestion
Refer to the [Bucketing Module example](https://github.com/apache/incubator-mxnet/blob/master/example/rnn/bucketing/cudnn_rnn_bucketing.py#L167)
```

> Refer to the [Bucketing Module example](https://github.com/apache/incubator-mxnet/blob/master/example/rnn/bucketing/cudnn_rnn_bucketing.py#L167)

Reviewer 4: Line numbers usually get obsolete and I have seen many broken links due to that. Can we just refer to cudnn_rnn_bucketing.py example and provide function name that users should read.

Author: Thanks for pointing out, addressed this
